google-cloud-speech/lib/google/cloud/speech/v1/speech_client_config.json

```diff
@@ -1,40 +1,40 @@
 {
   "interfaces": {
-    "google.cloud.speech.v1beta1.Speech": {
+    "google.cloud.speech.v1.Speech": {
       "retry_codes": {
-        "idempotent": [
-          "DEADLINE_EXCEEDED",
-          "UNAVAILABLE"
-        ],
-        "non_idempotent": [
-          "UNAVAILABLE"
-        ]
+          "idempotent": [
```

Comment:
@pbogle Please take a look at https://github.com/googleapis/googleapis/pull/270
Also please discuss with @shinfan regarding whether we should configure retry differently for speech.